lib/trento/clusters/events/host_added_to_cluster.ex

```diff
@@ -0,0 +1,12 @@
+defmodule Trento.Clusters.Events.HostAddedToCluster do
+  @moduledoc """
+  This event is emitted when a host is added to a cluster
+  """
+
+  use Trento.Event
+
+  defevent do
+    field(:cluster_id, Ecto.UUID)
```

Comment:
We shouldn't have parenthesis here. 
That's why git diff is a bit strange maybe